sources/iTermAdvancedSettingsModel.m

```diff
@@ -64,6 +64,8 @@ + (NSString *)name { \
 DEFINE_BOOL(allowDragOfTabIntoNewWindow, YES, @"Tabs: Allow a tab to be dragged and dropped outside any existing tab bar to create a new window.");
 
 #pragma mark Mouse
+DEFINE_STRING(alternateMouseScrollEscapeUp, @"", @"Mouse: Scroll wheel sends the specified escape sequence in alternate screen mode (up)");
```

Comment:
Add a newline and explain that the string should use vim syntax, such as `\e` for escape.